handlebars-i18next.es6

```diff
@@ -0,0 +1,23 @@
+const jsonKeys = ['lngs', 'fallbackLng', 'ns', 'postProcess', 'interpolation'];
+
+function extend(target, ...sources) {
```

Comment:
Maybe state why `Object.assign` hasn't been used here instead. Compatibility with older browsers?

Reply:
Exactly (IE11). Will add.

Or I could create a `lodash-object-assign` polyfill package and thell people to use that if they need it. But that's probably stretching it.

Reply:
I agree. If this were a bigger library I'd just expect people to be using some form of polyfill.